specification/billingbenefits/resource-manager/Microsoft.BillingBenefits/stable/2022-11-01/billingbenefits.json

```diff
@@ -0,0 +1,2224 @@
+{
```

Comment:
For new RP onboarding, must onboard to RpaaS
https://armwiki.azurewebsites.net/rpaas/overview.html

Reply:
Hi @NetRock I checked "new service onboarding" by mistake, this PR is only for "new API version" as we already have this RP onboarded last year. Please check the following private review PR of this RP's onboarding:
https://github.com/Azure/azure-rest-api-specs-pr/pull/4431
https://github.com/Azure/azure-rest-api-specs-pr/pull/5895
https://github.com/Azure/azure-rest-api-specs-pr/pull/6036
https://github.com/Azure/azure-rest-api-specs-pr/pull/7344
https://github.com/Azure/azure-rest-api-specs-pr/pull/9069

Also this service is GAed, you can find it here: https://azure.microsoft.com/en-us/pricing/offers/savings-plan-compute/